apps/docs-snippets/src/guide/types/arrays.test.ts

```diff
@@ -26,9 +21,23 @@ describe(__filename, () => {
   });
 
   it('should successfully execute echo u64 array contract call', async () => {
+    using launched = await launchTestNode({
+      contractsConfigs: [
+        {
+          deployer: EchoU64ArrayAbi__factory,
+          bytecode: EchoU64ArrayAbiHex,
+        },
+      ],
+    });
+
+    const {
+      contracts: [contract],
+    } = launched;
     // #region arrays-2
     const u64Array = [10000000, 20000000];
 
+    // This expects two arguments
+    // #TODO: Argument of type 'number[]' is not assignable to parameter of type '[BigNumberish, BigNumberish]'.
```

Comment:
Typegen mismatch 

```
Argument of type 'number[]' is not assignable to parameter of type '[BigNumberish, BigNumberish]'.
```

Reply:
This is expected. `u64Array` is inferred as a `number[]`. I believe that adding an `as const` assertion will solve it:
```ts
const u64Array = [10000000, 20000000] as const;
```
If not, we can do a cast:
```ts
const u64Array = [10000000, 20000000] as [BigNumberish, BigNumberish];
```

Reply:
This was a silly oversight on my part 😅  

1a486f24a